ros_ws/src/teleoperation/include/keyboard_controller.h

```diff
@@ -1,37 +1,82 @@
 #pragma once
 
-#include <ros/ros.h>
-
+#include <SDL.h>
 #include <drive_msgs/drive_param.h>
-#include <std_msgs/Int64.h>
-
+#include <ros/ros.h>
 #include <signal.h>
-#include <termios.h>
-#include <time.h>
 
 #define TOPIC_DRIVE_PARAMETERS "/set/drive_param"
 
+using std::min;
+using std::max;
+
 enum class Keycode : int
 {
-    W     = 119,
-    A     = 97,
-    S     = 115,
-    D     = 100,
+    W = 119,
+    A = 97,
+    S = 115,
+    D = 100,
     SPACE = 32
 };
 
+enum class KeyIndex : int
+{
+    ACCELERATE = 0,
+    DECELERATE = 2,
+    STEER_LEFT = 1,
+    STEER_RIGHT = 3
+};
+
+constexpr Keycode KEY_CODES[] = { Keycode::W, Keycode::A, Keycode::S, Keycode::D };
```

Comment:
This c-style array could be replaced by a `std::array`. This would allow "foreach"-loop e.g.
```C++
for(auto key : KEY_CODE){/* DO STUFF */}
```
which might me useful in the implementation of the `pollKeyboardEvents` member function.